database/dbService.js

```diff
@@ -0,0 +1,84 @@
+var mysql = require('mysql');
+var keys = require('../config/keys');
+var queries = require('./queries');
+
+var connection;
+module.exports = {
+    /* Inserts a given user model given by Google OAuth */
+    insertUser: function(user, callback) {
+        this.tryConnect().getConnection(function(err, con) {
+            var sql = queries.insertUser;
+            con.query(sql, [user.id, user.gender, user.name.givenName, user.name.familyName, user.emails[0].value], function (err, result) {
+                con.release();
+                if (err) throw err;
+                callback(result.insertId);
+            });
+        });
+    },
+
```

Comment:
The basic structure of these methods:
1) We connect to the database 
2) Make a SQL query (aka sql = queries.insertUser, in which queries.insertUser refers to a string of the query)
3) Make the query and get the response back from the database
4) Callback to the parent caller with this response